websites/R/RiseFM/presence.ts

```diff
@@ -0,0 +1,65 @@
+const presence = new Presence({
+  clientId: "773701779434897419"
+});
+
+
+let rname: string, rartist: string, listeners: string, islive: string, presenter: string, lTitle: string, yes: boolean = false;
+
+function metadataListener(): void {
+  const data = JSON.parse(this.responseText);
+  rname = data.now.title;
+  rartist = data.now.artist;
+  listeners = data.listeners.current;
+  islive = data.live.autoDJ;
+  presenter = data.live.userrname;
```

Comment:
wouldn't it be better solution to just use const?

Reply:
No he needs let here because he is setting it in a function and using it outside of the function

Reply:
right but he changed now things that SHOULD be const and merged